root/install-translations.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
```

Comment:
this file needs to be an init script dropped into `root/etc/cont-init.d` with the appropriate filename/number like here: https://github.com/linuxserver/docker-mods/blob/code-server-docker/root/etc/cont-init.d/98-docker